schema.sql

```diff
@@ -8,3 +8,23 @@ CREATE TABLE animals (
     species_id INTEGER REFERENCES species(id),
     owner_id   INTEGER REFERENCES owners(id)
 );
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY  NOT NULL,
+    full_name         TEXT NOT NULL,
+    age               INT  NOT NULL,
+)
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY  NOT NULL,
+    name              TEXT NOT NULL,
+)
+CREATE TABLE vets (
+    id                  SERIAL PRIMARY KEY,
+    name                TEXT,
+    age                 INTEGER,
+    date_of_graduation  DATE
+);
+CREATE TABLE specializations (
+    vet_id     INTEGER REFERENCES vets(id),
+    species_id INTEGER REFERENCES species(id),
+    PRIMARY KEY (vet_id, species_id)
+);
```

Comment:
- [ ] Just as you created this join table `specializations`, kindly consider creating another join table called `visits` that will complete the many-to-many relationship between vets and animals. `visits` will have vets(id) and animals(id) as references and also `visit_date date` for date of visit.
requirement: 
  - There is a many-to-many relationship between the tables animals and vets: an animal can visit multiple vets and one vet can be visited by multiple animals. Create a "join table" called visits to handle this relationship, it should also keep track of the date of the visit.